mod/metrics/lib/card/answer_query.rb

```diff
@@ -146,7 +142,7 @@ def run_filter_query
     def prepare_filter_args filter
       @filter_args = filter.deep_symbolize_keys
       # TODO: remove following (and handle consequences in tests!)
-      @filter_args[:latest] = true unless filter[:year] || filter[:metric_value]
+      # @filter_args[:latest] = true unless filter[:year] || filter[:metric_value]
```

Comment:
This is in some ways the biggest change.  Latest is still the default filter in the main html views, but not down deep in AnswerQuery.